systemtests/volsupervisor_test.go

```diff
@@ -140,27 +115,24 @@ func (s *systemtestSuite) TestVolsupervisorRestart(c *C) {
 
 	volName := genRandomVolume()
 	fqVolName := fqVolume("policy1", volName)
-
 	c.Assert(s.createVolume("mon0", fqVolName, map[string]string{"unlocked": "true"}), IsNil)
 
-	time.Sleep(4 * time.Second)
```

Comment:
What was the reason for this big change in sleep time?  I'm having trouble seeing it from the diff